src/helpers/menuConfigs.ts

```diff
@@ -86,18 +87,43 @@ function mapToLocalIcon<T extends { icon: string }>(
   }));
 }
 
-export const bundlerPackages = bundlerOptions.map(b => b.package);
+export const bundlerPackages = bundlerOptions.map((b) => b.package);
 
 export function getResourcePath(panel: vscode.WebviewPanel, baseUriResources: string, fileName: string) {
   return panel.webview.asWebviewUri(vscode.Uri.file(join(baseUriResources, fileName)));
 }
 
+export function getToolkitUri(webview: vscode.Webview, extensionUri: vscode.Uri) {
+  const pathList = ['node_modules', '@vscode', 'webview-ui-toolkit', 'dist', 'toolkit.min.js'];
+  return webview.asWebviewUri(vscode.Uri.joinPath(extensionUri, ...pathList));
+}
+
 export function getRepoTypeOptions(panel: vscode.WebviewPanel, baseUriResources: string) {
   return mapToLocalIcon(repoTypeOptions, panel, baseUriResources);
 }
 
+export async function getPiralTemplatesOptions() {
+  const baseUrl = 'https://registry.npmjs.org/-/v1/search?text=keywords:piral+template&size=250';
+  const result = await axios.get(baseUrl);
+  const piralTemplates = await result.data.objects.map((elm: any) => {
+    return elm.package.name;
+  });
+
+  return piralTemplates;
+}
+
+export async function getPiletTemplatesOptions() {
```

Comment:
The whole function is a duplicate of `getPiralTemplatesOptions`. Why don't you make one function such as `getTemplates` which is

```ts
async function getPiralTemplatesOptions(type: 'piral' | 'pilet', size = 50) {
  const baseUrl = `https://registry.npmjs.org/-/v1/search?text=keywords:${type}+template&size=${size}`;
  const result = await axios.get(baseUrl);
  const piralTemplates = await result.data.objects.map((elm: any) => {
    return elm.package.name;
  });

  return piralTemplates;
}
```

which is then called like this

```ts
export async function getPiralTemplates() {
  return getTemplates('piral');
}
```